Source/WebKit/UIProcess/Cocoa/ExtensionCapabilityGranter.mm

```diff
@@ -28,6 +28,8 @@
 
 #if ENABLE(EXTENSION_CAPABILITIES)
 
+#import <BrowserEngineKit/BECapability.h>
```

Comment:
This should go below the quoted imports. Also, we should import the umbrella header.